nornir/core/exceptions.py

```diff
@@ -1,6 +1,19 @@
 from builtins import super
 
 
+class ConnectionException(Exception):
+    def __init__(self, connection):
+        self.connection = connection
+
+
+class ConnectionAlreadyOpen(ConnectionException):
+    pass
+
+
+class ConnectionNotOpen(Exception):
+    pass
+
```

Comment:
```
class ConnectionNotOpen(ConnectionException): 
```
?

Reply:
Not sure what you are asking.

Reply:
if `ConnectionNotOpen` should inherit from `ConnectionException`?

Reply:
Right, that's a typo. Fixing.